1-js/01-getting-started/4-devtools/article.md

```diff
@@ -1,64 +1,64 @@
 
-Developer tools are potent; they have many features. To start, we'll learn how to open them, look at errors, and run JavaScript commands.
+Las herramientas de desarrollo son potentes; Tienen muchas características. Para comenzar, aprenderemos cómo abrirlas, observar errores y ejecutar comandos JavaScript.
 
 ## Google Chrome
 
-Open the page [bug.html](bug.html).
+Abre la página [bug.html](bug.html).
 
-There's an error in the JavaScript code on it. It's hidden from a regular visitor's eyes, so let's open developer tools to see it.
+Hay un error en el código JavaScript dentro de la página. Está oculto a los ojos de un visitante regular, así que abramos las herramientas de desarrollador para verlo.
 
-Press `key:F12` or, if you're on Mac, then `key:Cmd+Opt+J`.
+Presione la tecla `F12` o, si está en Mac, entonces la tecla`: Cmd + Opt + J`.
 
-The developer tools will open on the Console tab by default.
+Las herramientas de desarrollador se abrirán en la pestaña Consola de forma predeterminada.
 
-It looks somewhat like this:
+Se ve algo así:
 
 ![chrome](chrome.png)
 
-The exact look of developer tools depends on your version of Chrome. It changes from time to time but should be similar.
+El aspecto exacto de las herramientas de desarrollador depende de su versión de Chrome. Cambia de vez en cuando, pero debería ser similar.
 
-- Here we can see the red-colored error message. In this case, the script contains an unknown "lalala" command.
-- On the right, there is a clickable link to the source `bug.html:12` with the line number where the error has occurred.
+- Aquí podemos ver el mensaje de error de color rojo. En este caso, el script contiene un comando desconocido "lalala".
+- A la derecha, hay un enlace en el que se puede hacer clic en la fuente `bug.html: 12` con el número de línea donde se produjo el error.
 
-Below the error message, there is a blue `>` symbol. It marks a "command line" where we can type JavaScript commands. Press `key:Enter` to run them (`key:Shift+Enter` to input multi-line commands).
+Debajo del mensaje de error, hay un símbolo azul `>`. Marca una "línea de comando" donde podemos escribir comandos JavaScript. Presione la tecla `: Enter` para ejecutarlos (tecla`: Shift + Enter` para ingresar comandos de varias líneas).
 
-Now we can see errors, and that's enough for a start. We'll come back to developer tools later and cover debugging more in-depth in the chapter <info:debugging-chrome>.
+Ahora podemos ver errores, y eso es suficiente para empezar. Volveremos a las herramientas de desarrollador más adelante y cubriremos la depuración más en profundidad en el capítulo <info: debugging-chrome>.
 
 
-## Firefox, Edge, and others
+## Firefox, Edge, y otros
 
-Most other browsers use `key:F12` to open developer tools.
+La mayoría de los otros navegadores usan `tecla: F12` para abrir herramientas de desarrollador.
```

Comment:
creo que `key:F12`, no se traduce, es como `pattern:\b`, internamente el webserver interpreta a pattern, .... revisando: efectivamente no se traduce: [en ingles](https://javascript.info/devtools#firefox-edge-and-others)